src/cider/tests/nextgen/functional/CiderDateNextgenTest.cpp

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <gtest/gtest.h>
+#include "exec/nextgen/Nextgen.h"
+#include "exec/plan/parser/TypeUtils.h"
+#include "tests/utils/CiderTestBase.h"
+
+using namespace cider::exec::nextgen;
```

Comment:
As cheng's suggestion, let's simplify the test file name to `DateTest.cpp` when you do the rebase.